docs/workflows/react-native-specific-workflow-steps/index.md

```diff
@@ -56,7 +56,7 @@ This component provides detailed reports and insights on the results of React Na
 
 For detailed information on the usage of **Test Reports for React Native**, please refer to the documentation:
 
-[https://docs.appcircle.io/continuous-testing/react-native-testing](https://docs.appcircle.io/continuous-testing/react-native-testing)
+[/continuous-testing/react-native-testing](/continuous-testing/react-native-testing)
```

Comment:
IMO not seem professional:
![Screenshot 2024-11-01 at 17 05 47](https://github.com/user-attachments/assets/4abcdf26-f21d-4255-b9dc-d14dbcd23df3)

Can you update like taht:
```suggestion
[Appcircle React Native Testing](/continuous-testing/react-native-testing)
```

Reply:
➕ It should be fixed.